Source/Diagnostics/ReducedDiags/ParticleExtrema.cpp

```diff
@@ -487,27 +487,35 @@ void ParticleExtrema::ComputeDiags (int step)
             ParallelDescriptor::ReduceRealMax(chimax_f);
         }
 #endif
-        m_data[0]  = xmin;
-        m_data[1]  = xmax;
-        m_data[2]  = ymin;
-        m_data[3]  = ymax;
-        m_data[4]  = zmin;
-        m_data[5]  = zmax;
-        m_data[6]  = uxmin*m;
-        m_data[7]  = uxmax*m;
-        m_data[8]  = uymin*m;
-        m_data[9]  = uymax*m;
-        m_data[10] = uzmin*m;
-        m_data[11] = uzmax*m;
-        m_data[12] = gmin;
-        m_data[13] = gmax;
-        m_data[14] = wmin;
-        m_data[15] = wmax;
+        constexpr auto
+            idx_xmin   = 0  , idx_xmax   = 1   , idx_ymin   = 2 , idx_ymax   = 3,
+            idx_zmin   = 4  , idx_zmax   = 5   , idx_uxminm = 6 , idx_uxmaxm = 7,
+            idx_uyminm = 8  , idx_uymaxm = 9   , idx_uzminm = 10, idx_uzmaxm = 11,
+            idx_gmin   = 12 , idx_gmax   = 13  , idx_wmin   = 14, idx_wmax   = 15;
+        [[maybe_unused]] constexpr auto
+            idx_chimin_f = 16, idx_chimax_f = 17;
+
+        m_data[idx_xmin]  = xmin;
+        m_data[idx_xmax]  = xmax;
```

Comment:
I think this index change might take it too far for readability comparing to the left... what do you think?

It would be worth it if we use the index again at another location, but since we only use it here it looks a bit too confusing to me.

Reply:
You are right... is it better now ? otherwise I will revert to the initial state...